test/metabase/driver/mysql_test.clj

```diff
@@ -304,23 +310,26 @@
 (deftest group-on-time-column-test
   (mt/test-driver :mysql
     (testing "can group on TIME columns (#12846)"
-      (mt/dataset attempted-murders
-        (let [now-date-str (u.date/format (t/local-date (t/zone-id "UTC")))
-              add-date-fn  (fn [t] [(str now-date-str "T" t)])]
-          (testing "by minute"
-            (is (= (map add-date-fn ["00:14:00Z" "00:23:00Z" "00:35:00Z"])
-                   (mt/rows
-                     (mt/run-mbql-query attempts
-                       {:breakout [!minute.time]
-                        :order-by [[:asc !minute.time]]
-                        :limit    3})))))
-          (testing "by hour"
-            (is (= (map add-date-fn ["23:00:00Z" "20:00:00Z" "19:00:00Z"])
-                   (mt/rows
-                     (mt/run-mbql-query attempts
-                       {:breakout [!hour.time]
-                        :order-by [[:desc !hour.time]]
-                        :limit    3}))))))))))
+      (mt/with-temporary-setting-values [report-timezone "UTC"]
```

Comment:
This test failed from the REPL for me because my report timezone setting was set to something else.